### ISSUE TITLE
Training instructions unclear

### ISSUE THREAD

**Issue by maximecb** [lcswillems/rl-starter-files#23]
Hi Lucas,

The training instructions are a bit unclear. First you suggest to install torch-ac using pip. Then you suggest people run:

```
python3 -m scripts.train --algo ppo --env MiniGrid-DoorKey-5x5-v0 --model DoorKey --save-interval 10
```

There's a step missing which would be to clone your rl-starter-files repository.

Personally, I don't love the splitting into two repos. I find it complicates things a bit. Now there are two repos and more steps involved. You also have to keep in mind, it's likely that people will want to modify the PPO training code to add auxiliary losses or some such. I would encourage you to keep things in one repo and to remove levels of abstraction, reduce the amount of files and the amount of code, etc. Just my two cents.

**Comment by lcswillems** [comment]
Hi Maxime,

Thank you for the feedback.

It is already written in the README to start by cloning the repository, but I think I will bold it because it may not be visible enough.

There are also less steps currently.
Currently: 1 cloning, 1 command to execute `pip install gym-minigrid torch-ac`.
Previously: 2 cloning, several commands in each clone.

Finally, I splitted this repository into 2 because:

1. I needed to use `torch-rl` several times this year, and everytime, I removed the `torch_rl` part because it was already installed from a previous clone.

2. I wanted to create a PyPI package to make the use easier. Now, the package can easily be downloaded, upgraded, versioned, no need to execute hard-to-remember commands in hard-to-remember locations. For example, if I wanted to update `torch-rl` on my computer, I had to first find in which clone I first installed it.

3. I think it makes it clearer to understand what does what. Because both packages have different goals and can be used independently.

However, I understand it makes it harder to modify `torch-ac`, but I don't think it makes it much harder. We just need to clone `torch-ac` (or to fork then clone) and execute `pip install -e .`. Maybe should I had these commands in the README for people that would want to modify `torch-ac`?

Thank you for your feedback Maxime
